sea-orm-cli/src/migration.rs

```diff
@@ -1,49 +0,0 @@
-use clap::{App, AppSettings, Arg, SubCommand};
```

Comment:
You delete this file, but it used in `sea-orm-migration/src/cli.rs`

Reply:
You can see the problem here: https://github.com/SeaQL/sea-orm/runs/6401508328?check_suite_focus=true#step:6:259

Reply:
Sorry I forgot about this, I'll fix it in `sea-orm-migration` update